### DISCUSSION TITLE
Report Acknowledgement

### DISCUSSION THREAD

**Discussion by blackgrease** [blockscout/blockscout#9948] [Support]
Hi, can someone at the receiving end of security@blockscout.com acknowledge any security reports and or questions received? Just so it's evident the email(s) has arrived.

**Comment by ekawibisono** [comment]
hi @blackgrease what this email ? Where did you get it from?

**Reply by blackgrease** [reply]
I didnt receive any email. I sent the emails. The emails were reporting a security issue.
